src/main/java/io/github/eliux/mega/cmd/ExportInfo.java

```diff
@@ -52,4 +79,26 @@ public String getRemotePath() {
     public String getPublicLink() {
         return publicLink;
     }
+
+    public static boolean isValidDateFormat(String value) {
+        try {
+            LocalDateTime.parse(value, DateTimeFormatter.ofPattern(dateFormat, Locale.US));
```

Comment:
Also, instead of checking whether the format is valid or not, what you could do is just to create a parsing function that returns an `Optional`. Also before parsing the date, you should probably specify the parameter `isValidDateFormat` so you make sure that you where explicit about the format you are getting the date from. So if in the future they change the default format the library does not break.